packages/cli/src/consoleParamsReader.ts

```diff
@@ -0,0 +1,121 @@
+import yargs from 'yargs';
+import {Config, Field} from "../../shared/type";
+import {askUserForFieldValue} from "./consoleWizzard";
+import {isValidDate} from "./validators";
+import {expectNever} from "../../server/src/util/util";
+import {Either, isLeft, left, right} from "fp-ts/lib/Either";
+
+const NO_WIZARD_LABEL:string = "noWizard";
+
+const addType = (field: Field, yargs: yargs.Argv) => {
+    if (field.list) {
+        yargs.array(field.name);
+        return;
+    }
+    switch (field.type) {
+        case "semver": // intentional fallthrough since semver isn't supported by yargs
+        case "date": // intentional fallthrough since date isn't supported by yargs
+        case "ffversion": // intentional fallthrough since ffversion isn't supported by yargs
+        case "string": {
+            yargs.string(field.name);
+            return;
+        }
+        case "boolean": {
+            yargs.boolean(field.name);
+            return;
+        }
+        case "number": {
+            yargs.number(field.name);
+            return;
+        }
+        default: {
+            expectNever(field.type)
+        }
+    }
+};
+
+const addDescription = (fields: Field[], yargs: yargs.Argv) => {
+    let description = {};
+    description[NO_WIZARD_LABEL] = "Prevents the wizard from starting when fields aren't set. Options are: \n" +
+        "t - fills all missing required fields with the needed data type as placeholder\n" +
+        "to - fills all missing required and optional fields with the needed data type as placeholder\n" +
+        "if no option is provided, the fields will simply be missing, but must be added per hand to obtain a valid change log";
+    fields.forEach(field => description[field.name] = field.description ? field.description : '');
+    yargs.describe(description);
```

Comment:
Weniger forEach und mutability :D.
```suggestion
    const interactiveDescription = "Prevents the wizard from starting when fields aren't set. Options are: \n" +
        "t - fills all missing required fields with the needed data type as placeholder\n" +
        "to - fills all missing required and optional fields with the needed data type as placeholder\n" +
        "if no option is provided, the fields will simply be missing, but must be added per hand to obtain a valid change log";
    const description = fields.reduce((all, field) => ({...all, [field.name]: field.description ?? ''}), {});
    description[NO_WIZARD_LABEL] = interactiveDescription;
    yargs.describe(description)
```